modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -22,6 +22,9 @@ function acquia_cms_headless_install($is_syncing) {
 
     // Adds the necessary permissions to roles.
     add_next_site_permissions();
+
+    // Install the restui module.
+    \Drupal::service('module_installer')->install(['restui']);
```

Comment:
@apathak18 this doesn't make any sense to remove restui from info and installing it from .install instead just leave restui in info.yml and remove other module, what do you say?

cc @vishalkhode1

Reply:
We are switching restui to a soft dependency, which allows customers to remove it if not required. Aligning with best practices, UI-related modules should typically be uninstalled in production environments to enhance performance.

Reply:
Agree with @rajeshreeputra.